docs/modules/ROOT/pages/appcat/availability.adoc

```diff
@@ -0,0 +1,19 @@
+= Zone Availability of Services
+
+.Table Availability
```

Comment:
```suggestion
```

Please add some table styling and table semantics (`cols`, `options=header`, ...) and I'd leave the title away

Reply:
Oh, I see, it's just a draft... Never mind